CMake/cdat_modules/pep8_pkg.cmake

```diff
@@ -0,0 +1,18 @@
+set( PEP8_MAJOR 1 )
+set( PEP8_MINOR 6 )
+set( PEP8_PATCH 2)
+set( PEP8_VERSION ${PEP8_MAJOR}.${PEP8_MINOR}.${PEP8_PATCH} )
+set( PEP8_URL ${LLNL_URL} )
+set( PEP8_GZ pep8-${PEP8_VERSION}.tar.gz )
+set( PEP8_MD5 c7a3f57d832484a6295164661fbb1335 )
+
+set (nm PEP8)
+string(TOUPPER ${nm} uc_nm)
+set(${uc_nm}_VERSION ${${nm}_MAJOR}.${${nm}_MINOR}.${${nm}_PATCH})
+set(PEP8_SOURCE ${PEP8_URL}/${PEP8_GZ})
+
+if (CDAT_BUILD_ALL)
+  add_cdat_package(pep8 "" "" ON)
+else()
+  add_cdat_package(pep8 "" "" OFF)
```

Comment:
it should be OFF even in ALL mode. ALL is meant for users and PEP is a developer tool. In near future, I would like to test ALL as well and more you will add to ALL, it will make it difficult to test it on dashboard machines. 

I like the idea and wanted to talk about it.